packages/web/components/Form/Repeater.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useFieldArray, useForm } from 'react-hook-form';
+
+const Repeater = ({ childsComponent, endComponent, name, emptyValue, initialValue }) => {
+	const { control } = useForm({
```

Comment:
Só tem um detalhe aqui, esse formulário vai ser diferente do formulário do `FormWizard`, pois o `FormWizard` já define um form e expõe ele através da `prop` form, esse default value aqui é obrigatório? Outra opção é ver se tem como pegar um `form` já definido e setar os `defaultValues`

Reply:
vou dar uma olhada 👍

Reply:
Ok, passei o "form" para baixo via props, com relação a alterar o defaultValues, eu precisaria resetar o form

![Screenshot 2020-06-21 at 12 54 36](https://user-images.githubusercontent.com/8526488/85223915-66ea2300-b3be-11ea-963f-2c73b079d862.png)

Eu achei mais simples criar um `useEffect` e fazer append de um elemento na criação do component, o que acha?

Reply:
Acho que dá certo assim! Fica mais simples mesmo!

Reply:
Depois de mudar o `useForm()` para `form={form}` meus unit tests pararam de funcionar, passei um bocado de tempo tentando resolver o problema mas não consegui, ao rodar os testes que estavam funcionando antes recebo o erro `cannot read 'control' of undefined` devido a essa linha https://github.com/ufersa/plataforma-sabia/pull/231/files#diff-ddbd941eced9cc0e2268c4f99bdbe5e6R9 a msg de error tenta induzir a pensar que o erro é na **linha 7** https://github.com/ufersa/plataforma-sabia/pull/231/files#diff-ddbd941eced9cc0e2268c4f99bdbe5e6R7 entretanto não é, é algo na implementação do `useFieldArray`, para reproduzir o erro caso você tenha interesse é só remover o skip dos testes que você irá ver, uma possível solução seria mockar essa lib, entretanto os testes precisariam ser refeitos pois como mockaria a lib, a maioria das funções precisariam ser mockadas `(append, remove etc)`.